Document browser support (#1204)
silx-kit/h5web: packages/app/README.md

- [Create React App v4](https://codesandbox.io/s/h5webapp-cra-v4-0y1lj)
- [Create React App v5](https://codesandbox.io/s/h5webapp-cra-v5-bzmbh1)

### Browser support

H5Web works out of the box on **Firefox 78 ESR**.

Support for Firefox 68 ESR is possible by polyfilling the `ResizeObserver` API.

> One easy way to do this is with [polyfill.io](https://polyfill.io/v3/):

Reviewer: ```suggestion
[polyfill.io](https://polyfill.io/v3/) is an easy way to do this:
```

```html
<head>
  <!-- title, meta, link, etc. -->

>   <script src="https://polyfill.io/v3/polyfill.min.js?features=default%2CResizeObserver"></script>

Author: I went with `polyfill.io` in the end, as it's the simplest solution. It adds a request, but the response is empty if everything is already available. Polyfilling manually with `@juggle/resize-observer` or the like is more difficult to do right.

Reviewer: I think that's great :+1:
